frontend/src/components/PLAYGROUND/ReportSelector/index.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from "react";
+import {
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Card,
+  CardContent,
+  Button,
+  CardActions,
+  Typography,
+} from "@mui/material";
+import { Field, FormikProvider, useFormik } from "formik";
+import { categoriesAndTypes } from "./reportSelectorConfig";
+import Grid from "@mui/material/Unstable_Grid2";
```

Comment:
haha, love the file name.

Reply:
hahaha that's funny didn't even notice it.